test/version-tolerance/generated/llc-initial/README.md

```diff
@@ -0,0 +1,96 @@
+# LLC client library for JavaScript
+
+LLC Swagger, this is the initial swager a service could do
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Package (NPM)](https://www.npmjs.com/package/@msinternal/rlcClient)
+- [Samples](https://github.com/Azure-Samples/azure-samples-js-management)
```

Comment:
why are you generate sample links for mgmt plane SDK here?

Reply:
Updated